client/state/posts/test/reducer.js

```diff
@@ -114,7 +147,7 @@ describe( 'reducer', () => {
 				}
 			} );
 		} );
-		it( 'never persists state because this is not implemented', () => {
+		it( 'never persists state', () => {
```

Comment:
We can persist this provided that the selectors can handle items and sitePosts potentially not being in sync.